moon/cfc_chat_transit/server/init.moon

```diff
@@ -76,4 +77,39 @@ ChatTransit.ReceiveMessage = (ply, text, teamChat) =>
 
     @Logger\debug "Sent message '#{text}' to websocket"
 
+ChatTransit.PlayerConnected = (ply) =>
+
+    steamName = ply\Nick!
+    steamId = ply\SteamID64!
+    irisId = "none"
```

Comment:
You can omit `irisId` if you don't plan on using it in your go code

Reply:
Should i remove irisId in ReceiveMessage aswell, considering it isn't used?